docs/_static/auth/iam.yaml

```diff
@@ -637,6 +639,30 @@ components:
           type: string
           description: The session ID that can be used to retrieve the access token by the calling application.
           example: "eyJhbGciOiJSUzI1NiIsI"
+    UserDetails:
+      type: object
+      description: |
+        Claims about the user, to be provided when not using an OpenID Connect Identity Provider.
+      required:
+        - name
+        - id
+        - role
+      properties:
+        name:
+          type: string
+          description: Human-readable name of the user.
+          example: "Wouter de Bruijn"
```

Comment:
existing person...

Reply:
Copilot came up with it...

`Example van der Example` better? :'-)